Dockerfile

```diff
@@ -1,12 +1,12 @@
-# Use the latest Ubuntu image as the base
-FROM ubuntu:rolling
+# Use the latest slim Debian unstable image as the base
+# TODO: Move to stable when buster is out (sid is needed for QEMU 3.1)
+FROM debian:unstable-slim
```

Comment:
I think we'd prefer testing over unstable. Is there any reason you picked unstable?

Reply:
apt.llvm.org does not have testing repos, just old stable, stable, and unstable. I think testing base + unstable LLVM works but that's fragile to me (testing base + stable LLVM does not work).

Reply:
I run my workstation with a testing base and unstable/experimental packages along side. I do this as there's a lower risk of a broken package showing up.

However, your argument makes just as much sense as mine. I'll leave the choice to you.

Reply:
I agree. I will change this when I get home. If there is ever a dependency issue, we can just upgrade to sid then.